### PR TITLE
:pencil: Improve README content

### PR DESCRIPTION
**Pull request by kefranabg**
Small README improvements 😃 

README generated with [https://github.com/kefranabg/readme-md-generator](https://github.com/kefranabg/readme-md-generator)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,58 @@
...
+Please ⭐️ this repository if you like it.
+
+## 📝 License
+
+Copyright © 2019 [Zenika](https://github.com/zenika-open-source).
+
+This project is [Apache 2.0](https://github.com/zenika-open-source/insights-website/blob/master/LICENSE) licensed.
+
+***
+_This README was generated with ❤️ by [readme-md-generator](https://github.com/kefranabg/readme-md-generator)_
```
Comment (kefranabg) [inline README.md]: I can remove this footer if you want 😉

File: package.json [hunk 2]
```diff
@@ -26,7 +26,8 @@
   "keywords": [
     "gatsby"
   ],
-  "license": "MIT",
+  "homepage": "https://zenika-open-source-insights.netlify.com",
```
Comment (bpetetot) [inline package.json]: Now we have a better URL ;)
https://oss.zenika.com

File: README.md (hunk 1 above)
Comment (bpetetot) [reply README.md]: No, keep it, it's fair ;)

File: README.md [hunk 3]
```diff
@@ -1,19 +1,58 @@
...
+</p>
 
-# insights-website
+> White label website to display awesome insights about organization open source activity
 
-White label website to display awesome insights about organisation open source activity
+## ✨Homepage
 
-## Install
+[https://zenika-open-source-insights.netlify.com](https://zenika-open-source-insights.netlify.com)
```
Comment (bpetetot) [inline README.md]: Now we have a better URL ;)
https://oss.zenika.com

File: README.md [hunk 4]
```diff
@@ -1,19 +1,58 @@
...
+  <a href="https://circleci.com/gh/zenika-open-source/insights-website/tree/master">
+    <img alt="CircleCI" src="https://circleci.com/gh/zenika-open-source/insights-website/tree/master.svg?style=svg" target="_blank" />
+  </a>
+  <a href="https://app.netlify.com/sites/zenika-open-source-insights/deploys">
+    <img alt="Netlify Status" src="https://api.netlify.com/api/v1/badges/44effe10-3635-426d-899d-984201fa828a/deploy-status" target="_blank" />
+  </a>
+</p>
 
-# insights-website
+> White label website to display awesome insights about organization open source activity
```
Comment (bpetetot) [inline README.md]: @zenika-open-source/insights-website should we keep "White label" while it's not one for now and maybe for a moment?
Instead it should be better to have an issue : "Make it a white label"

**Pull request merged**